### DISCUSSION TITLE
Create3 robot turns white light but never start up

### DISCUSSION THREAD

**Discussion by Alice-devpe** [iRobotEducation/create3_docs#576] [Questions]
### How are you connecting to your Create 3?

Wi-Fi and USB Ethernet (ROS 2)

### Which version of ROS 2 is installed on your computer?

Humble

### Which firmware version is installed on your robot?

H.2.6

### Which RMW is your robot running?

FastRTPS

### Does your robot have an assigned namespace? If so, please share.

No

### Is the robot connected to a network? If so, what is the network type?

Yes: laboratory

### Are there multiple Create 3 robots connected to your network?

No

### Is multicast enabled?

No

### What is the Adapter Board's USB/BLE Toggle currently switched to?

USB

### Describe your question.

When i put the Create3 robot in the charge station, the white light turns on and begins to blinking, but never finish to boot, it stays booting up, and doesn't make any sound. I tried with RP4 connected, and without, with Adapter board also in USB and BLE too, but nothing change. Is there something or a way to reset it?

**Comment by alsora** [comment]
Could you share the logs page from the webserver? https://iroboteducation.github.io/create3_docs/webserver/logs/

To turn off the robot you should do a long press of the main button when the robot is not on the dock.
Alternatively, remove the bottom screws and take out the battery and put it back in.
